ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.spec.ts

```diff
@@ -232,43 +206,40 @@ describe('ActivitiPeople', () => {
             jasmine.Ajax.uninstall();
         });
 
-        it('should log error message when search fails', async(() => {
+        it('should log error message when search fails', () => {
             activitiPeopleComponent.peopleSearch$.subscribe(() => {
                 expect(logService.error).toHaveBeenCalledWith('Could not load users');
             });
             activitiPeopleComponent.searchUser('fake-search');
             jasmine.Ajax.requests.mostRecent().respondWith({
                 status: 403
             });
-        }));
+        });
 
-        it('should not remove user if remove involved user fail', async(() => {
-            activitiPeopleComponent.people.push(fakeUser);
-            fixture.detectChanges();
+        it('should not remove user if remove involved user fail', () => {
```

Comment:
i don't think the test is testing correctly the behaviour without the async clause here.